test/ref_ops_test.cpp

```diff
@@ -2439,6 +2439,100 @@ TEST_CASE(logsoftmax_test_axis_3)
     EXPECT(migraphx::verify_range(results_vector, s));
 }
 
+TEST_CASE(loop_test)
```

Comment:
Can we put the ref loop unit tests in a seperate cpp file?

Reply:
changed